### ISSUE TITLE
Working with changeable roi_num

### ISSUE THREAD

**Issue by giladdiv** [yhenon/keras-frcnn#210]
I am trying to use this model ,where ROI number need to be changed at each iteration.
The only problem I found to do this change is the FOR loop in RoiPoolingConv,py
how can I replace the for loop(that need a constant number for the iteration number) with a loop that dont have a fix number of iterations(that will use K.shape(num_rois)[0]).
 
`        for roi_idx in range(self.num_rois):

            x = rois[0, roi_idx, 0]
            y = rois[0, roi_idx, 1]
            w = rois[0, roi_idx, 2]
            h = rois[0, roi_idx, 3]
            
            row_length = w / float(self.pool_size)
            col_length = h / float(self.pool_size)

            num_pool_regions = self.pool_size

            #NOTE: the RoiPooling implementation differs between theano and tensorflow due to the lack of a resize op
            # in theano. The theano implementation is much less efficient and leads to long compile times
            if self.dim_ordering == 'tf':
                x = K.cast(x, 'int32')
                y = K.cast(y, 'int32')
                w = K.cast(w, 'int32')
                h = K.cast(h, 'int32')

                rs = tf.image.resize_images(img[:, y:y+h, x:x+w, :], (self.pool_size, self.pool_size))
                outputs.append(rs)

        final_output = K.concatenate(outputs, axis=0)
        final_output = K.reshape(final_output, (1, self.num_rois, self.pool_size, self.pool_size, self.nb_channels))`
